requirements.txt

```diff
@@ -41,3 +41,6 @@ colorama==0.4.4
 # Building config files interactively
 questionary==1.10.0
 prompt-toolkit==3.0.20
+
+#Futures
+schedule==1.1.0
```

Comment:
if we need a new dependency, please add it to `setup.py` (and environment.yml) as well ...

Reply:
It's imported into `freqtradebot` so unless we create a subclass called `FuturesBot` or something, then it will always need to be installed, so I don't think any changes would need to be made to `setup.sh`, because the script already installs everything from `requirements.txt`

Reply:
it doesn't appear to be a heavy dependency (difficult to install, whatever other dependencies) - so going the effort to properly isolate it won't be necessary here.

I'm however not talking about `setup.sh`, but `setup.py`.

while we recommend users to install frozen requirements first (`pip install requirements.txt`) - no-one guarantees users follow that.
you can also run `pip install freqtrade` - which will ignore the content of `requirements.txt`, and use whatever is defined in `setup.py` .